reporting_app/__init__.py

```diff
@@ -240,7 +240,7 @@ def report_samples(view_type):
     if view_type == 'all':
         title = 'All samples'
         ajax_call['api_urls'] = [
-            util.construct_url('samples', max_results=10000),
+            util.construct_url('samples', max_results=15000),
```

Comment:
Will this cause a problem when we go over 15000 samples?

Reply:
Yes, very much so.
A better solution should be found by then.
I'll raise an issue for it.